sympy/polys/matrices/domainmatrix.py

```diff
@@ -1690,3 +1690,62 @@ def unify_eq(A, B):
         if A.domain != B.domain:
             A, B = A.unify(B)
         return A == B
+
+    def lll(A, delta=QQ(3, 4)) -> 'DomainMatrix':
```

Comment:
If we use `from __future__ import annotations` then I guess we don't need to use a string annotation.